asgard/projector.h

```diff
@@ -100,8 +118,27 @@ class Projector {
         return results;
     }
 
-    size_t get_nb_cache_miss() const { return nb_cache_miss; }
-    size_t get_nb_calls() const { return nb_calls; }
+    template<typename T>
+    std::unordered_map<std::string, valhalla::baldr::PathLocation>
+    project_without_cache(const T places_begin,
+                          const T places_end,
+                          valhalla::baldr::GraphReader& graph,
+                          const std::string& mode,
+                          const valhalla::sif::cost_ptr_t& costing) const {
+        std::vector<valhalla::baldr::Location> locations;
+        for (auto it = places_begin; it != places_end; ++it) {
+            locations.push_back(build_location(*it, reachability, radius));
+        }
```

Comment:
very-minor: For the sack of modern code :D 
```suggestion
        std::transform(places_begin, places_end, std::back_inserter(locations), 
              bind(build_location, placeholders::_1, reachability, radius));
```